docs/articles/user_acquisition_analytics.md

```diff
@@ -0,0 +1,442 @@
+)
+umap_df = umap_df.join(label_df)
+```
+
+Next,we join our dataframes and create a chart, letting us visualize the UMAP-transformed vectors, and coloring them with cluster labels.
+
+```python
+umap_df = umap_df.join(label_df)
+alt.Chart(umap_df).mark_circle(size=8).encode(
+    x="dimension_1", y="dimension_2", color="cluster_label:N"
+).properties(
+    width=600, height=500, title="UMAP Transformed vectors coloured by cluster labels"
+).configure_title(
+    fontSize=16,
+    anchor="middle",
+).configure_legend(
+    strokeColor="black",
+    fillColor="#EEEEEE",
+    padding=10,
+    cornerRadius=10,
+    labelFontSize=14,
+    titleFontSize=14,
+).configure_axis(
+    titleFontSize=14, labelFontSize=12
+)
+```
+
+![cluster visualization](../assets/use_cases/user_acquisition_analytics/cluster_visualization.png)
+
+
+The dark blue clusters (label -1) are outliers - not large or dense enough to form a distinct group. 
+The large number of blobs [better word?] results from the fact that 1/3 of the vector norm mass is made of not very many ad_creatives.
+Note - 2D (UMAP) visualizations often make some clusters look quite dispersed [scattered].
+
+## Understanding the cluster groups
+
+To understand our user clusters better, we can generate some activity histograms.
+
+First, we join user data with cluster labels, create separate DataFrames for each cluster, generates activity histograms for each cluster, and then concatenates these histograms into a single visualization.
+
+```python
+# activity histograms by cluster
+user_df = user_df.set_index("id").join(label_df)
+
+by_cluster_data = {
+    label: user_df[user_df["cluster_label"] == label]
+    for label in np.unique(hdbscan.labels_)
+}
+
+activity_histograms = [
+    alt.Chart(user_df_part)
+    .mark_bar()
+    .encode(x=alt.X("activity", bin=True), y="count()")
+    .properties(title=f"Activity histogram for cluster {label}")
+    for label, user_df_part in by_cluster_data.items()
+]
+
+alt.hconcat(*activity_histograms)
+```
+
+![histograms of clusters](../assets/use_cases/user_acquisition_analytics/cluster_histograms.png)
+
+From our histograms, we can observe that:
+
+- cluster -1 has an unusually large number of active users (typically, there are *very* few highly active application users)
+- cluster 2 and cluster 3 users are quite similar, but low activity
+- cluster 0 has the highest proportion of medium activity users
+- cluster 1 users are moderately active, with relatively few very high activity users and very few low activity users
+
+To see the distribution of ad_creatives across different clusters, we create a DataFrame that shows each ad_creative's count value within each cluster:
+
+```python
+pd.DataFrame(user_df.groupby("cluster_label")["ad_creative"].value_counts())
+```
+
+![ad creatives distribution across clusters](../assets/use_cases/user_acquisition_analytics/ad_creatives-per-cluster.png)
+
+observations:
```

Comment:
@morkapronczay
This needs updating in the notebook and google colab.
As does the summary table at the bottom of the colab.